plugins/services/src/js/utils/FrameworkUtil.js

```diff
@@ -76,6 +83,52 @@ const FrameworkUtil = {
     }
 
     return splitVersion.slice(1).join("-");
+  },
+
+  /**
+   * Return a warning or an error label
+   * if the pacakge has cosmosPackage been updated soon.
+   * @param {object} cosmosPackage The cosmos package to check
+   * @returns {null | React.ReactNode} null or a label saying when was the last update of this package
+   */
+  getLastUpdated(cosmosPackage) {
+    // This looks like magic, but we are only decoding the package configuration into a string,
+    // and then finding the PACKAGE_BUILD_TIME_STR property.
+    const v2AppMustacheTemplate = Util.findNestedPropertyInObject(
+      cosmosPackage,
+      "marathon.v2AppMustacheTemplate"
+    );
+    if (!v2AppMustacheTemplate) {
+      return null;
+    }
+
+    const decodedBuildTime = atob(v2AppMustacheTemplate).match(
+      /PACKAGE_BUILD_TIME_STR": "(.*)"/
+    );
+
+    if (!decodedBuildTime) {
+      return null;
+    }
+
+    const packageUpdatedTime = decodedBuildTime[1];
+    const now = Date.now();
+
+    const difference = now - DateUtil.strToMs(packageUpdatedTime);
+    if (difference < SIX_MONTHS) {
+      return null;
+    }
+
+    const className = difference > YEAR ? "danger" : "warning";
```

Comment:
I decided to show a yellow label for packages older than six months, but younger than a year and a red label for packages older that a year. Again, the designers might change this.